ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -67,7 +74,11 @@ let tdb_tool = !Xapi_globs.tdb_tool
 
 let domain_krb5_dir = Filename.concat Xapi_globs.samba_dir "lock/smb_krb5"
 
-let debug_level () = !Xapi_globs.winbind_debug_level |> string_of_int
+let debug_level =
```

Comment:
This is now evaluated when the module (from this ml file) loads when the process starts. Previously, this was a function in order to delay the evaluation of `!Xapi_globs.winbind_debug_level`, which gets initialised from the config file when `Xapi_main` loads. This is a race condition, because the order in which modules are loaded in not predictable.